examples/article/complex.article.yaml

```diff
@@ -0,0 +1,44 @@
+type: Article
+title: Introducing eLife’s first computationally reproducible article
+url: https://elifesciences.org/labs/ad58f08d/introducing-elife-s-first-computationally-reproducible-article
+authors:
+  - Giuliano Maciocci
+  - Michael Aufreiter
+  - Nokome Bentley
+content:
+  - type: Paragraph
+    content:
+      - type: '#text'
```

Comment:
We no longer use `type: '#text'`. This item in `content` should be just a plain string.